### ISSUE TITLE
⌨ Insufficient contrast - Accordion subtitle

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#6898]
## ⌨ Accessibility Issue

The accordion subtitle has insufficient contrast.

## 📋 Steps to Reproduce

1. Navigate to /en/search
2. Run an Axe scan

## 📸 Screenshot

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/8499f6c1-0021-434d-a60a-18b7c45ee799)

## 🙋‍♀️ Proposed Solution

It looks like later designs have the subtitle grey anyway.

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/293d6234-8245-4499-a39f-71cdbd7f23c7)

## ✅ Acceptance Criteria

- [ ] Accordion subtitles meet the expected contrast ratio of 4.5:1

**Comment by petertgiles** [comment]
@substrae Can you confirm the subtitles on accordions should be grey?

**Comment by substrae** [comment]
@petertgiles 

Yep, grey away!
